pkg/cloudprovider/providers/vsphere/vsphere.go

```diff
@@ -881,6 +883,11 @@ func (vs *VSphere) AttachDisk(vmDiskPath string, storagePolicyID string, nodeNam
 		return "", "", err
 	}
 	return deviceName, diskUUID, nil
+	}
+	requestTime := time.Now()
+	diskID, diskUUID, err = attachDiskInternal(vmDiskPath, storagePolicyID, nodeName)
+	recordvSphereMetric(request_attachvolume, requestTime, err)
+	return diskID, diskUUID, err
```

Comment:
I don't know if this was intended, but this metric doesn't just capture attach disk time - but it captures vsphereLogin and other bunch of calls too. 

I think, it is fine to capture that metric too - but then is not really single api call latency but more like time it took for entire operation. Is that the intention here?